### DISCUSSION TITLE
Help with an android install crashing every time it starts

### DISCUSSION THREAD

**Discussion by G1TN0** [slackhq/nebula#1232] [Q&A]
Hi,

Long time fan of nebula, first time I've had any problems and I'm stumped...  Any help greatly appreciated.

I have nebula running on a few android devices for various things but by most recent project has got me a little bit stuck.  
I have installed nebula on a Samsung Galaxy A13, imported the certificates and set the IPs etc. for the lighthouse.   Every time I run the app on the A13 phone I get this error below... 

"Something went wrong with Nebula
Nebula closed because this app has a bug.
Try updating this app after its developer provides a fix for this error."

I'm using Android version 14, patch level 1st July 2024 and Samsung One UI version 6.1
App version 0.4.1-77 (sha: 7a048d8)
Nebula version 1.8.1 (go1.20.20)
Flutter version  3.3.5
Dart version 2.18.2

I've tried the usual of uninstalling and re-installing.  I've also factory reset the phone as I only want nebula and one other app to run on it.

I don't suppose there are any other versions of the APK available to see if an earlier version works?  

I'm running the same version on my Galaxy fold and it has worked perfectly since day one.   

Thanks for any help.

**Comment by IanVS** [comment]
@G1TN0 thanks for the report.  I've been updating the app recently and am nearly ready to release a new test version.  If you'd like to test out the pre-release, email your email address to info@defined.net, and I'll add you to our list of test users.  And if that doesn't fix the issue you're having, I'll dig in further.

**Comment by IanVS** [comment]
I've copied this issue to https://github.com/DefinedNet/mobile_nebula/issues/173 to track it there.
